File: BaiduMapTest/IPLocationTest.cs
using BaiduMap.Request;
using BaiduMap.Request.Models;
using System;
using System.Collections.Generic;
using System.Text;
using Xunit;
using Shouldly;

namespace BaiduMapTest
{
    public class IPLocationTest
    {
        [Fact]
        public void Test()
        {
            var client = ClientCreator.Create();
            var model = new IPModel()
            {
                //IP = "183.134.216.113"
            };
            var req = new IPLocationRequest(model);
            var resp = client.Execute(req);
            resp.Address.ShouldNotBeNull();
            resp.Content.ShouldNotBeNull();
        }
    }
}

File: JAVA/baidumap-webapi/src/test/java/wang/seamas/baidumap/IPLocationTest.java
package wang.seamas.baidumap;

import org.junit.Assert;
import org.junit.Test;
import wang.seamas.baidumap.request.IPLocationRequest;
import wang.seamas.baidumap.request.model.IPModel;
import wang.seamas.baidumap.response.IPLocationResponse;

/**
 * @author: Seamas Wang
 * @description:
 * @date: Created in 11:22 2018/8/6
 */
public class IPLocationTest {

    @Test
    public void test() {
        BaiduApiClient client = ClientCreator.Create();

        IPModel model = new IPModel();
        IPLocationRequest request = new IPLocationRequest(model);
        IPLocationResponse response = client.execute(request);
        Assert.assertEquals(response.getStatus(), 0);
    }
}
